_posts/2022-10-05-pdf-generation-at-scale.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "PDF generation at scale with OpenFaaS and Puppeteer"
+description: "We show how to use OpenFaaS with back pressure and autoscaling to generate PDFs with puppeteer"
```

Comment:
Learn how to scale headless browsers to massive scale with Puppeteer and OpenFaaS.